src/integrationTest/resources/tests/arm/properties/live/case-record.properties

```diff
@@ -0,0 +1,10 @@
+bf_001=OBJECT_TYPE
+bf_002=CASE_NUMBERS
+bf_003=FILE_TYPE
+bf_005=CHECKSUM
+bf_009=COMMENTS
+bf_010=CREATED_DATE_TIME
+bf_012=OBJECT_ID
+bf_013=PARENT_ID
+bf_016=UPLOADED_BY
+bf_019=COURTHOUSE
```

Comment:
Looks like we're missing `bf_020=COURTROOM` in live?

Reply:
Case document is not associated with a courtroom

Reply:
OK, can see you have removed the nle one in commit https://github.com/hmcts/darts-api/commit/fdba1dad9b936d14aebb4bbc5cefe286cbc3f5d3

Reply:
case doesn't have courtroom